example-jest/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@reportportal/agent-js-jest-example",
+  "version": "1.0.0",
+  "description": "Example for @reportportal/agent-js-jest",
+  "repository": {
```

Comment:
Please remove redundant fields from here.
I suggest to leave: necessary dependecies, scripts.
Also will be good to add node version support info:
```
  "engines": {
    "node": ">= 8.0.x"
  },
```